schemas/metaschema/Domain.schema

```diff
@@ -13,7 +12,9 @@
   format: { type: 'string' },
   min: { type: 'number' },
   max: { type: 'number' },
-  signed: { type: 'boolean' },
   bytes: { type: 'number' },
-  comment: { type: 'string' }
+  comment: { type: 'string' },
+  parse: { type: 'function' },
+  serialize: { type: 'function' },
+  normalize: { type: 'function' },
 }
```

Comment:
Add validation function as well?

Reply:
We have `validate` field in `DatabaseField` but domains may not be linked to record, so do not support calculation and validation because we have no place in code to dynamically execute this schema. I'll move `normalize` to `DatabaseField` too @nechaido

Reply:
@tshemsedinov it can be used when we validate an object against a category, form or anything else, see https://github.com/metarhia/metaschema/pull/154/files#diff-f412f4722a52cf22ed3d0a9a694b837dR187.